### DISCUSSION TITLE
Confused about installling codesnap plugin

### DISCUSSION THREAD

**Discussion by dsriggs1** [nix-community/nixvim#1972] [Q&A] [answered]
I can't seem to configure out the appropriate way to enable the codesnap plugin within nixvim. It looks like it is supported in the documentation here: https://nix-community.github.io/nixvim/plugins/codesnap/index.html

Below is the relevant part of my configuration that I am using for other nixvim plugins, but with codesnap I am getting the error message:

"The option `home-manager.users.user.programs.nixvim.plugins.codesnap-nvim' does not exist"

```
{
  programs.nixvim = {
    plugins.codesnap-nvim = {
      enable = true;
    };
  };
}
```

Any help is greatly appreciated.

**Comment by GaetanLepage** [comment]
Hey !
The option is `plugins.codesnap`, not `plugins.codesnap-nvim`.
This is often the way we name plugin modules.

**Reply by dsriggs1** [reply]
Hey! Thanks for the response this the error I am getting when switching to plugins.codesnap.

error: The option `home-manager.users.user.programs.nixvim.plugins.codesnap' does not exist. Definition19:27:47 [3/882]
       - In `/nix/store/4m6pykj6lnw1s8minw38dm0m8bjxqwsr-source/neovim/codesnap.nix':
           {
             enable = true;
           }

**Reply by GaetanLepage** [reply, marked as answer]
Are you using the stable or unstable version of nixvim ?
Maybe `codesnap` is only available in the `unstable` branch.
